apps/graphql/src/types/output/Passenger.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLInt,
+  GraphQLBoolean,
+} from 'graphql';
+
+export default new GraphQLObjectType({
+  name: 'Passenger',
+  description: 'Passenger details',
+  fields: {
+    bags: {
+      type: GraphQLInt,
+    },
+    birthday: {
+      type: GraphQLString,
+    },
+    category: {
+      type: GraphQLString,
+    },
+    firstname: {
+      type: GraphQLString,
+    },
+    id: {
+      type: GraphQLInt,
```

Comment:
Can you make this a global id instead?

Reply:
https://github.com/kiwicom/margarita/blob/master/apps/graphql/src/types/output/BookingInterface.js#L16

Reply:
Ok, changed to global id now